docs/_includes/header-wide.html

```diff
@@ -15,6 +15,12 @@
 <link rel="icon" type="image/png" href="/dist/images/android-chrome-192x192.png" sizes="192x192">
 <link rel="icon" type="image/png" href="/dist/images/favicon-96x96.png" sizes="96x96">
 <link rel="icon" type="image/png" href="/dist/images/favicon-16x16.png" sizes="16x16">
+
+<!-- Dark mode favicon -->
+<link rel="icon" href="/dist/images/favicon-dark-mode-16x16.png" type="image/x-icon" media="(prefers-color-scheme: dark)" sizes="16x16">
+<link rel="icon" href="/dist/images/favicon-dark-mode-32x32.png" type="image/x-icon" media="(prefers-color-scheme: dark)" sizes="32x32">
+<link rel="icon" href="/dist/images/favicon-dark-mode-96x96.png" type="image/x-icon" media="(prefers-color-scheme: dark)" sizes="96x96">
+
 <link rel="shortcut icon" type="image/x-icon" href="/dist/images/favicon.ico">
 <meta name="msapplication-TileColor" content="#204f79">
 <meta name="msapplication-TileImage" content="/dist/images/mstile-144x144.png">
```

Comment:
This cheeky lad is still triangles https://deploy-preview-325--id7.netlify.app/dist/images/mstile-144x144.png